_posts/2020-12-06-converting-your-laptop-to-a-janus-modem-using-unetaudio.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: post
+comments: true
+title: Converting your laptop to a JANUS modem using Unet audio
+date: 06/12/2020
+author: Manu Ignatius, Mandar Chitre
+categories: howto
+feature-img: "assets/img/unetaudio/unetaudio.jpg"
+thumbnail: "assets/img/unetaudio/unetaudio.jpg"
+tags: [howto, unetaudio, unetstack, modems, phy, janus]
+---
+
+Imagine you are developing an application for an underwater use case such as message or file transfer and you intend to eventually deploy the app on a network of [JANUS](http://www.januswiki.com/) compliant modems in the field. Or you may be developing a new routing protocol that is intended to work on a network of JANUS compliant modems. Or you might be a university Professor designing an exercise for your students to learn about underwater communications and networking.
```

Comment:
```suggestion
Imagine you are developing an application for an underwater use case such as messaging or file transfer and you intend to eventually deploy the app on a network of [JANUS](http://www.januswiki.com/) compliant modems in the field. Or you may be developing a new routing protocol that is intended to work on a network of JANUS compliant modems. Or you might be a university Professor designing an exercise for your students to learn about underwater communications and networking.
```

Reply:
Nitpicking here, but technically speaking, one can't really do a routing protocol on top of JANUS right? AFAIK JANUS doesn't support carrying any payload right? I remember there this whole `cargo` bit. But I don't remember if we support it.

Reply:
We support cargo, but the spec is unclear on how one should do it. So the implementation can have interop issues. But other than that, we can support everything from the upper layers over JANUS as long as you make the frame long enough to deal with the payload headers.

Reply:
Then will go with the current text.